### PR TITLE
Update the Rendersearchpage to filter the location by readius(miles).

### PR DESCRIPTION
**Pull request by soni653**
## Description ##

DESCRIBE YOUR PULL REQUEST'S CHANGES AND REFERENCE THE FEATURE/USERSTORY/BUG IT'S REALATED TO.  REPLACE THIS TEXT.

## Type ##

- [ ] This is a bugfix.
- [x] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [x] Is this a work in progress?
- [ ] Is this ready to be reviewed?
- [ ] Have you added two reviewers to this pull request?

### CODE REVIEW

File: src/components/pages/Search/RenderSearchPage.js [hunk 1]
```diff
@@ -28,16 +52,29 @@ const Searching = () => {
   }, []);
 
   const handleChange = event => {
+    console.log('getting the event:', event);
     setSearchValue(event.target.value);
   };
 
+  // const milesChange = value => {
```
Comment (cearc-sara) [inline src/components/pages/Search/RenderSearchPage.js]: make sure if you leave this code in here, that you leave a comment next to all the commented out code stating what they are used for. If they do not need to be there, then delete all the commented out code.

**Review by cearc-sara — CHANGES_REQUESTED** [review]
Please review the comments in the code, and make the appropriate changes.

File: src/components/pages/Search/RenderSearchPage.js (hunk 1 above)
Comment (soni653) [reply src/components/pages/Search/RenderSearchPage.js]: Hi Sara , i deleted all commented out code and submited new PR.

**Review by cearc-sara — APPROVED** [review]
Looks good now.

**Review by vishalicious213 — APPROVED** [review]
My only concern is not knowing why the start & build scripts have been changed to include --max_old_space_size=4096. The rest seems fine.

**Pull request merged**